src/core/border_router/routing_manager.cpp

```diff
@@ -1111,7 +1111,7 @@ void RoutingManager::HandleRouterSolicit(const Ip6::Address &aSrcAddress,
     }
 #endif
 
-    // Schedule Router Advertisements with random delay.
+    // Schedule routing policy evaluation with random delay.
```

Comment:
```suggestion
    // Schedule routing policy evaluation with random jitter to respond with Router Advertisement
```
nit: document the purpose of routing policy evaluation

Reply:
Updated.